README.md

```diff
@@ -108,18 +108,30 @@ Option 1: Pull down the image from [Docker Hub](https://hub.docker.com/r/qasimwa
 
 `docker pull qasimwarraich/cli-tutor`
 
+`podman pull qasimwarraich/cli-tutor`
+
 Note: You can run the container using:
+
 `docker run -it qasimwarraich/cli-tutor`
 
+`podman run -it qasimwarraich/cli-tutor`
+
 ---
 
 Option 2: Building image without make
 
+>⚠️ Make sure a **Dockerfile** present, otherwise the build process will fail.⚠️
```

Comment:
Don't think this is necessary, I am also using the name `Containerfile` now.

Reply:
Yea. I Saw that. Removing this.